bin/manager_start.sh

```diff
@@ -48,5 +48,4 @@ else
   $appdir/spark-job-server.jar $2 $3 $4 $conffile'
 fi
 
-eval $cmd > /dev/null 2>&1 &
-# exec java -cp $CLASSPATH $GC_OPTS $JAVA_OPTS $LOGGING_OPTS $CONFIG_OVERRIDES $MAIN $1 $2 $conffile 2>&1 &
+eval $cmd
```

Comment:
can you add a blank line at the end of the file?

Reply:
@zeitos Nicely spotted. Just did that.